velox/functions/lib/aggregates/tests/AggregationTestBase.h

```diff
@@ -206,6 +206,15 @@ class AggregationTestBase : public exec::test::OperatorTestBase {
         });
   }
 
+  /// Use abandon-partial-aggregation optimization to trigger the toIntermediate
+  /// code path.
+  void testFailingAggregations(
+      const std::vector<RowVectorPtr>& data,
+      const std::vector<std::string>& groupingKeys,
+      const std::vector<std::string>& aggregates,
+      const std::unordered_map<std::string, std::string>& config = {},
+      const std::string& expectedMessage = "");
```

Comment:
Let's move this error message so that it doesn't have a default value. There is no point in calling this method without specifying the error message.

One option is to move expectedMessage to be the first argument. Another to move it just before config.

Reply:
Fixed.